DISTRIBUTIONS.md

```diff
@@ -149,6 +149,7 @@
 - [kubetail](https://github.com/jbvmio/kubetail/): Repository for kubetail - The pod log tailer
 - [kubeval](https://github.com/instrumenta/kubeval/): Validate your Kubernetes configuration files, supports multiple Kubernetes versions
 - [kustomize](https://github.com/kubernetes-sigs/): Customization of kubernetes YAML configurations
+- [linkerd](https://github.com//linkerd/linkerd2/):Linkerd is an ultralight, security-first service mesh for Kubernetes. Linkerd adds critical security, observability, and reliability features to your Kubernetes stack with no code change required.
```

Comment:
Hi @Sierra1011 ,
There is a space missing 
```
:Linkerd
 ^
```

Reply:
:cry:

Reply:
thank you @Sierra1011 :)